Start a11y-profile-manager-indicator only if not running already && Fixes issues starting orca patch fixing style issues.

--- source/ubiquity/frontend/gtk_ui.py.orig	2022-05-24 06:56:47.305813027 +0000
+++ source/ubiquity/frontend/gtk_ui.py	2022-05-24 06:56:50.509845089 +0000
@@ -166,7 +166,8 @@
         # setting gets passed to the target system.
         if (gsettings.get_boolean(key) and osextras.find_on_path('orca')):
             # Enable
-            subprocess.Popen(['orca', '--replace'], preexec_fn=misc.drop_all_privileges)
+            subprocess.Popen(['orca', '--replace'],
+                             preexec_fn=misc.drop_all_privileges)
             os.environ['UBIQUITY_A11Y_PROFILE'] = 'screen-reader'
         elif 'UBIQUITY_A11Y_PROFILE' in os.environ:
             # Disable
@@ -391,7 +392,8 @@
         self.customize_installer()
 
         # Put up the a11y indicator.
-        if osextras.find_on_path('a11y-profile-manager-indicator') and (subprocess.run(['pgrep', '-f', 'a11y-profile-manager-indicator'], capture_output=True).returncode == 0):
+        if osextras.find_on_path('a11y-profile-manager-indicator') and (subprocess.run(
+                ['pgrep', '-f', 'a11y-profile-manager-indicator'], capture_output=True).returncode == 0):
             try:
                 subprocess.Popen(['a11y-profile-manager-indicator',
                                   '-i'], preexec_fn=misc.drop_all_privileges)
@@ -1125,7 +1127,9 @@
         self.allow_go_backward(False)
 
         misc.add_connection_watch(self.network_change)
-        misc.add_connection_watch(self.set_connectivity_state, global_only=False)
+        misc.add_connection_watch(
+            self.set_connectivity_state,
+            global_only=False)
 
     def set_window_hints(self, widget):
         if (self.oem_user_config or
